ui/src/components/app-form/app-form.tsx

```diff
@@ -1,5 +1,5 @@
+import ErrorIcon from '@mui/icons-material/Error';
```

Comment:
This should probably be this: `import ErrorRoundedIcon from '@mui/icons-material/ErrorRounded';`

Reply:
Replaced
<img width="667" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/610d317c-e055-4593-ac24-059037aa16bb">
<img width="102" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/2587009b-fbbf-4a22-96ae-56324778a0ed">